spec/models/user_spec.rb

```diff
@@ -16,4 +16,30 @@
   it 'has a valid factory' do
     expect(subject).to be_valid
   end
+
+  describe '#full_name' do
+    let(:first_name) { Faker::Name.neutral_first_name }
+    let(:last_name) { Faker::Name.last_name }
+    let(:user) { create(:user, first_name: first_name, last_name: last_name) }
+
+    it 'returns the full name of the user' do
+      expect(user.full_name).to eq("#{first_name} #{last_name}")
+    end
+
+    it 'handles missing first name' do
+      user[:first_name] = ''
+      expect(user.full_name).to eq(last_name)
+    end
```

Comment:
Estos tests deberían tener una estructura de contexto
```suggestion
    context 'when the ...' do
      let(:first_name) { '' }
      # it ...
    end
```